static/templates/bookmarks.html

```diff
@@ -6,7 +6,7 @@
     </span>
     <span class="bookmarkFooter">
         <img class="bookmarkFavicon" src="{{favicon}}" />
-        <a href="{{url}}" class="bookmarkUrl">{{url}}</a>
+        <a href="{{url}}" data-id="{{id}}" class="bookmarkUrl">{{url}}</a>
```

Comment:
Let's set it on the root of bookmark, so you don't have to use `.find` twice as on line 132-133